x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/counted_keyword/10_basic.yml

```diff
@@ -49,3 +66,20 @@ setup:
   - match: { aggregations.event_terms.buckets.2.key: "c" }
   - match: { aggregations.event_terms.buckets.2.doc_count: 2 }
   - length: { aggregations.event_terms.buckets: 3 }
+
+# although the field is not indexed, the counted_terms agg should still work
+  - do:
```

Comment:
Maybe also verify that no inverted index is created for the `events` field in `test-event-no-index` index? Maybe by executing a field caps api call and/or disk usage api call and verify the response?

Reply:
Good point! I've also split the test case files for the default options and for `index: false` as the latter will be introduced in 8.13 + it keeps the two cases neatly separated.